csharp-basics/exercises/TypesAndVariables/Exercise 5/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿class Program
+{
+    static void Main()
+    {
+        string table = $@"+------------------------------------------------------------+
+| 1 |                          English III |       Ms. Lapan |
```

Comment:
Use several variables to store the names of your classes and their teachers. Tu neesi izpildījis uzd pēc noteikumiem

Reply:
Tulīt ielikšu pareizo variantu